concent_api/utils/api_view.py

```diff
@@ -81,6 +81,7 @@ def wrapper(request, *args, **kwargs):
             )
             return JsonResponse({'error': str(exception)}, status = 400)
         if isinstance(response_from_view, Message):
+            assert response_from_view.sig is None
```

Comment:
This is already in master, please remove it and rebase on master if you need it.